### ISSUE TITLE
Monaco editor breaks when Orchard Core is deployed to a subdirectory

### ISSUE THREAD

**Issue by TaylorSpindle** [OrchardCMS/OrchardCore#11434]
### Describe the bug
When an Orchard Core instance is deployed with a subdirectory via IIS Monaco Editor is broken. This is part of the loader not honoring the subdirectory. Please see below example where TaxPayer is the subdirectory and all other scripts load correctly but not Monaco:

### To Reproduce
1. Deploy Orchard Core test website to IIS to a subdirectory 
2. Open Admin Screen
3. Go to content type and set HTML Editor as Monaco editor
4. attempt to edit or add content that uses the Monaco editor

### Expected behavior
The expected behavior is that the Monaco editor should honor the subdirectory that it is deployed to not strip them out. 

### Screenshots
![image](https://user-images.githubusercontent.com/102378520/160165883-d2525106-333e-46ef-8d96-e6809e5233c3.png)
![image](https://user-images.githubusercontent.com/102378520/160164857-bc95eac7-6deb-4728-a4ca-975e87667724.png)

**Comment by jptissot** [comment]
is your tenant / site configured with the subdirectory as a UrlPrefix ?

**Comment by TaylorSpindle** [comment]
Yes, that is correct. It is configured via IIS with a URL Prefix. I tried setting the Base Url in Configuration -> Settings ->General. This did not affect the error. Thank you for your help.

**Comment by jtkech** [comment]
The tenant prefix was already taken into account but not a possible virtual folder that would be not part of the tenant prefix but part of the pathBase of a given request, It has been fixed by #11370.

But maybe not yet available from nuget, maybe only from the cloudsmith feed.

**Comment by hyzx86** [comment]
You can add this element to your child tenant's 'Layout' file
``` HTML
<base href="https://example.com/tenant1">
```

**Comment by sebastienros** [comment]
Closing as fixed.
